src/networkextension.cs

```diff
@@ -162,13 +163,19 @@ interface NEAppProxyFlow {
 		[Export ("closeWriteWithError:")]
 		void CloseWrite ([NullAllowed] NSError error);
 
+		[Internal]
 		[NoWatch, NoTV, NoiOS, Mac (10,15,4)]
 		[Export ("setMetadata:")]
-		void SetMetadata (OS_nw_parameters parameters);
+		void SetMetadata (OS_nw_parameters nwparameters);
+
+		[NoWatch, NoTV, NoiOS, Mac (10,15,4)]
+		[Wrap ("SetMetadata (parameters.Handle)")]
```

Comment:
NRE ;-) use `GetHandle ()`

Reply:
lol, too much cmd+z to file https://github.com/xamarin/xamarin-macios/issues/8127 🤣

Reply:
```suggestion
		[Wrap ("SetMetadata (parameters.GetHandle ())")]
```